debian/control.workstation

```diff
@@ -0,0 +1,9 @@
+Package: securedrop-workstation-grsec
+Section: admin
+Architecture: ${DEBARCH}
+Pre-Depends: qubes-kernel-vm-support (>=4.0.31)
+Depends: linux-image-${PACKAGE_VERSION}, libelf-dev, paxctld
```

Comment:
I think we can also drop libelf-dev?

Reply:
After some spelunking, I think so. Based on <https://github.com/freedomofpress/securedrop-builder/pull/2> it was only needed for the building of the Qubes kernel modules, which we thankfully don't do anymore.